scania-digital-design-system/tegel
# fix: align input text colors

## **Describe pull-request**  
This PR aligns the text colors of labels, placeholders and input text for Text Field, Textarea and Dropdown components. These alignments are not present in the UI Library yet but have been verified with a designer. For testing reference, the following is considered true:

- Input text in **light mode** should be _grey-958_
- Input text in **dark mode** should be _grey-50_
- Labels (both inside and outside position) in **light mode** should be _grey-958_
- Labels (both inside and outside position) in **dark mode** should be _grey-50_
- Placeholders in **light mode** should be _grey-500_
- Placeholders in **dark mode** should be _grey-600_

## **Issue Linking:**  
- **Jira:** [CDEP-3563](https://tegel.atlassian.net/browse/CDEP-3563)    

## **How to test**  
1. Go to Storybook link
2. Check in Text Field, Textarea and Dropdown
3. Verify that the above parts of components have the correct values

## **Checklist before submission**

## **Suggested test steps**


[CDEP-3563]: https://tegel.atlassian.net/browse/CDEP-3563?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
